### PR TITLE
Attempt docs fix

### PR DESCRIPTION
**Pull request by JonathonMisiewicz**
## Description
Attempts to restore the docs. Previously, we were getting warnings:
```
2022-03-10T17:32:55.6518847Z /home/runner/work/psi4/psi4/code/objdir/doc/sphinxman/source/notes_c.rst:66: ERROR: Unexpected indentation.
2022-03-10T17:32:55.6519492Z /home/runner/work/psi4/psi4/code/objdir/doc/sphinxman/source/notes_c.rst:67: WARNING: Block quote ends without a blank line; unexpected unindent.
2022-03-10T17:32:55.6520109Z /home/runner/work/psi4/psi4/code/objdir/doc/sphinxman/source/notes_c.rst:70: WARNING: Bullet list ends without a blank line; unexpected unindent.
```

Wouldn't GHA docs auto-build be nice...?

## Status
- [x] Ready for review
- [x] Ready for merge

### CODE REVIEW

File: doc/sphinxman/source/notes_c.rst [hunk 1]
```diff
@@ -60,20 +60,23 @@ Notes on Psivars
...
+   * ``method ROOT 0 (h) -> ROOT m (i) property`` to get the transition
+     between two roots, specifying the symmetry of both states and the index of the target
+     roots among states of their own symmetry.
+
+   For example, to target the second excited-state, which is also the lowest energy state
+   of its irrep, the first two calls will take m = 2, while the last takes m = 0.
+   Methods that use this interface are: TD-fctl.
+   Note that numberings are associated with the calculation much more strongly than 
+   with the molecular system. Changing the number of roots sought, the symmetry 
+   subspace or the symmetry apportionment or roots under which the computation is run,
```
Comment (loriab) [inline doc/sphinxman/source/notes_c.rst:78]:
```suggestion
   subspace or the symmetry apportionment of roots under which the computation is run, 
```
my typo from suggestion on other PR

File: CMakeLists.txt [hunk 2]
```diff
@@ -309,6 +309,7 @@ include(ConfigTesting)
 
 # <<<  Install  >>>
 
+
```
Comment (loriab) [inline CMakeLists.txt:312]:
```suggestion
```
if you want

**Review by loriab — APPROVED** [review]

**Review by loriab — APPROVED** [review]

File: doc/sphinxman/source/notes_c.rst [hunk 3]
```diff
@@ -60,20 +60,23 @@ Notes on Psivars
 ================
 
 .. note:: Starting in 1.6, there are three standard ways to access an excited state
-    property. We give examples below, but the method name and property name may change.
-    * ``method ROOT 0 -> ROOT m property`` to get root m.
-    * ``method ROOT 0 -> ROOT m property - h TRANSITION`` to get root m and
+   property. We give examples below, but the method name and property name may change.
+   * ``method ROOT 0 -> ROOT m property`` to get root m.
+
+   * ``method ROOT 0 -> ROOT m property - h TRANSITION`` to get root m and
```
Comment (zachglick) [inline doc/sphinxman/source/notes_c.rst:66]: Should `A2` be replaced with `h` in this sentence? And then maybe mention that a possible value of `h` is `A2`?

**Review by zachglick — APPROVED** [review]

File: doc/sphinxman/source/notes_c.rst [hunk 4]
```diff
@@ -60,20 +60,23 @@ Notes on Psivars
...
       independently specify that the total transition symmetry is A2.
```
Comment (JonathonMisiewicz) [inline doc/sphinxman/source/notes_c.rst:67]:
```suggestion
      independently specify that the total transition symmetry is h, e.g., A2.
```

**Review by maxscheurer — APPROVED** [review]

**Pull request merged**